### ISSUE TITLE
Issues with next version of ggplot2

### ISSUE THREAD

**Issue by thomasp85** [stefanedwards/lemon#28]
Hi

We preparing the next release of ggplot2 and our reverse dependency checks show that your package is failing with the new version.

You can install the release candidate of ggplot2 using `devtools::install_github('tidyverse/ggplot2@v3.4.0-rc')` to test this out.

We plan to submit ggplot2 by the end of October and hope you can have a fix ready before then

Kind regards
Thomas

**Comment by NowakDG** [comment]
Hi @stefanedwards,

I think I am experiencing an error with lemon package (0.4.5) after updating to ggplot2 (3.4.0). I really enjoy using your package and I wonder if you plan to address this in the near future?

Best wishes,
Dawid

```
# example
library(ggplot2)
library(lemon)
p <- ggplot(mtcars, aes(x = mpg)) + 
  geom_dotplot() +
  theme_bw() +
  theme(panel.border=element_blank(), axis.line=element_line())
p1 <- p + lemon::coord_capped_cart(left="both")
p1  
```
Error in if (guides$x$position == position) { : 
    argument is of length zero

**Comment by stefanedwards** [comment]
I'm really sorry, but my package broke with the latest update to ggplot2 v.
3.4.0, and I haven't had the time to address any of the issues.
If you are adventurous, you are welcome to take a crack at it -- it's open
source, after all.

Again, my apologies


Den tor. 10. nov. 2022 kl. 15.29 skrev Dawid G. Nowak <
***@***.***>:

**Comment by stefanedwards** [comment]
Thanks

These issues are fixed with the release of version 0.4.6 (pending approval on CRAN) with commit fd42de399b60fb72070700437042a8ca9e6f0511.

Try installing this version with `devtools::install_github('stefanedwards/lemon')`.
